test/solidity/Facets/HopFacetPackedL2.t.sol

```diff
@@ -0,0 +1,329 @@
+pragma solidity 0.8.17;
+
```

Comment:
All of my comments on test/solidity/Facets/HopFacetPackedL1.t.sol seem to apply here, too